src/booleans.js

```diff
@@ -1,61 +1,61 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return (a&&b);
```

Comment:
The formatting is a bit out here, you should have some spaces around the variables and operators to make this cleaner to read:
```
  return a && b;
```